### ISSUE TITLE
Accessibly on WebGL/3D interactive element rules

### ISSUE THREAD

**Issue by NBM-Megan** [w3c/wcag#4731]
Accessibly on WebGL/3D interactive element rules

Would like to know what the regulations be for these new technologies that cannot be test by HTML

**Comment by patrickhlauke** [comment]
WCAG is not HTML-specific. it's technology-agnostic for any content that is classed as "web content". so unless certain aspects simply don't exist for a particular technology, all WCAG SCs already apply as they are.
